crates/telio-lana/src/lib.rs

```diff
@@ -161,16 +156,25 @@ pub fn init_moose(
     }
     moose::init(
         event_path,
-        LANA_APP_NAME.to_string(),
-        app_version,
-        LANA_MOOSE_VERSION.to_string(),
         prod,
         Box::new(moose_callbacks::MooseInitCallback {
             init_success_tx: tx,
         }),
         Box::new(moose_callbacks::MooseErrorCallback),
-    )
+    )?;
+    moose::set_context_application_version(app_version)?;
+    moose::set_context_application_name(LANA_APP_NAME.to_string())
 }
+
+/// Fetches context string from moose
```

Comment:
I am slightly confused here:
- why do we need to fetch the context anymore at all
- this `String` I think is JSON, shouldn't it be JSON structure then?

Reply:
Because we store meshnet_id in moose context.
And it is a string not a JSON, because this function receives the path through json to meshnet_id string.

Reply:
Where do we store the meshnet ID in moose by the way(and why do we need to re-retrieve it if we already have it)? I want to understand the flow better :) thanks!

Reply:
To be able to store it persistently. If the we restart meshnet we want to use the same meshnet id, not to generate a new one.